src/main.jsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
+import Root from "./routes/route";
+import ErrorPage from "./error-page";
 import './index.css'
 
+const router = createBrowserRouter([
```

Comment:
you have called the `createBrowserRouter` but you need to import it from somewhere to be able to use it